### PR TITLE
Created the blog post announcing Data Prepper 2.0

### PR DESCRIPTION
**Pull request by dlvenable**
### Description

We are releasing Data Prepper 2.0.0 on Oct 10. This is our announcement blog post.

This requires the bio for @oeyh as supplied in #1067.
 
### Issues Resolved

N/A

### Check List
- [x] Commits are signed per the DCO using --signoff
- [x] Update authors list after the bio is available from @oeyh.
- [ ] Merge in #1067 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by dlvenable** [comment]
I added Hai to the list of authors based on the username he supplied in #1067. We will need to merge that in prior to this PR.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:102]: Suggested rewrites for directory structure section:

--- 

Before the release of Data Prepper 2.0, we distributed Data Prepper as a single executable JAR file. While convenient, it made it difficult for us to include custom plugins.

We now distribute Data Prepper 2.0 in a bundled directory structure. This structure features a shell script to launch Data Prepper and dedicated subdirectories for JAR files, configurations, pipelines, logs, and more.

```
data-prepper-2.0.0/
  bin/
    data-prepper                    # Shell script to run Data Prepper
  config/
    data-prepper-config.yaml        # The Data Prepper configuration file
    log4j.properties                # Logging configuration
  pipelines/                             # New directory for pipelines
    trace-analytics.yaml
    log-ingest.yaml
  lib/
    data-prepper-core.jar
    ... any other jar files
  logs/
```
You can launch Data Prepper by running `bin/data-prepper`; no need for additional command line arguments or Java system property definitions. Instead, the application loads from the `config/` subdirectory.

Data Prepper 2.0 reads pipeline configurations from the `pipelines/` subdirectory. You can now define pipelines across multiple YAML files in the subdirectory, where each file contains the settings for one or more pipelines. The directory also helps keep pipeline definition distinct and, therefore, more compact and focused.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:143]: Sections edits:

---


Data Prepper 2.0 includes several other improvements. We want to highlight a few of them.

* The OpenSearch sink now supports any `create` actions for OpenSearch when writing or updating documents. Pipeline authors can configure their pipelines to create new documents when one does not already exist.
* The HTTP source now supports loading TLS/SSL credentials from either Amazon S3 or Amazon Certificate Manager. The OTel Trace Source supported these options; pipeline authors can now configure them for their log ingestion use cases.
* Data Prepper now requires Java 11, and the Docker image deploys with JDK 17.

Please see our release notes for a complete list.

**Review by Naarcha-AWS — APPROVED** [review]
Added my rewrites for each section in the review. One comment = one section. 

Might need to wait to add documentation links until this PR is merged: https://github.com/opensearch-project/documentation-website/pull/1510

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:12]: Or maybe:

The Data Prepper maintainers are proud to announce the release of Data Prepper 2.0. This release makes Data Prepper easier to use and helps you improve your observability stack based on feedback from you, our users.

Here are some of the major changes and enhancements made for Data Prepper 2.0.

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:12]: @dlvenable: Could we add a line in this intro or somewhere in the blog about OpenSearch compatibility? Data Prepper 2.0 is compatible with all OpenSearch versions, correct?

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:143]: > This change also allows you to import CSV or TSV formatted files from Amazon S3 sources, which is helpful for systems like Amazon CloudFront, which write their access logs as TSV files.

The word "which" was repeated so I modified this somewhat.

**Comment by dlvenable** [comment]
Thanks @Naarcha-AWS ! I took most of the changes to all sections except the Directory Structure. I want to check with @oeyh on those first.

I did make some tweaks from your suggestions - most of them were to try to be more accurate.

I also wasn't quite sure about some of the paragraphs. Did you intend all those paragraphs? The ones in the examples read too broken up and didn't keep the same train of thought.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md]: I added the following:

> Data Prepper 2.0 retains compatibility with all current versions of OpenSearch.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:113]: I added "You can _now_ launch Data Prepper..." to clarify this is new.

Also, "application loads _configurations_ from"

And I used definition here: "contains the _definition_ for one or more pipelines." I think settings is not quite clear.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:22]: "Use-case" should be "use case," per our style guidelines. Change to the following:
```suggestion
One common use case for conditional routing is reducing the volume of data going to some clusters.
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:23]:
```suggestion
When you want info logs that produce large volumes of data to go to a cluster, index with more frequent rollovers, or add deletions to clear out large volumes of data, you can now configure pipelines to route the data with your chosen action.
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:28]:
```suggestion
Then for any sink that should only have some data coming through, define one or more routes to apply. Data Prepper will evaluate 
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:29]:
```suggestion
these expressions for each event to determine which sinks to route these events. Any sink that has no routes defined will accept all events.
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:43]: Let's break this up a little more:

---

The pipeline then uses the `grok` processor to split the log line into multiple fields. The `grok` processor adds a named `loglevel` to the event. Pipeline authors can use that field in routes.

This pipeline contains two OpenSearch sinks. The first sink will only receive logs with a log level of `WARN` or `ERROR`. Data Prepper will route all events to the second sink.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:76]:
```suggestion
There are many other use cases that conditional routing can support. If there are other conditional expressions 
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:83]: Is there a distinction between "peer-forwarder" and peer forwarder? Should this be:

```suggestion
Previous to Data Prepper 2.0, performing stateful trace aggregations required using the peer forwarder processor plugin. 
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:146]:
```suggestion
Many of our users have incoming data with embedded JSON or CSV fields. To help in these use cases, Data Prepper 2.0 
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:156]: To reduce the number of times, we use "which," we can change it in this instance to "that". This might get changed in again in editorial, so while I'm leaving my suggestion I'll defer to @natebower on how best to avoid repetition here.

```suggestion
CSV or TSV fields, Data Prepper 2.0 now contains a `csv` processor that can create fields from your incoming CSV data.
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:163]:
```suggestion
* The HTTP source now supports loading TLS/SSL credentials from either Amazon S3 or Amazon Certificate Manager. Pipeline authors can now configure them for their log ingestion use cases. Before Data Prepper 2.0, only the OTel Trace Source supported these options.
```

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (Naarcha-AWS) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:164]:
```suggestion
* Data Prepper now requires Java 11 or higher. The Docker image deploys with JDK 17.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
A few more minor tweaks before we pass them off to @natebower.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:28]: Line breaks should not affect the rendered page.

Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md]: There was a space and line break which did create a new paragraph in the rendered page. Thanks for noting that!

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md]: I took your suggestion and made one clarification by adding "field" which you can see here: "... adds a 
_field_ named `loglevel` ..."

**Comment by dlvenable** [comment]
It took all the suggested changes.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (natebower) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:168]: The only thing we're missing here is a call to action. We need to conclude with a couple sentences telling the reader what we'd like for them to do next or where they can go to learn more. The below is an example from a recent blog post announcing Snapshot Management (SM):

**Wrapping it up**

SM automates taking snapshots of your cluster and provides useful features like notifications. To learn more about SM, check out the [SM documentation section](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-management/). For more technical details, read the [SM meta issue](https://github.com/opensearch-project/index-management/issues/280).

If you’re interested in snapshots, consider contributing to the next improvement we’re working on: [searchable snapshots](https://github.com/opensearch-project/OpenSearch/issues/3895).

**Review by natebower — COMMENTED** [review]
@dlvenable Please see my changes and comments, and let me know if you have any questions. Thanks!

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md]: I'm not sure about the change to "states" here. Using a singular noun for state is quite common.

> In information technology and computer science, a system is described as stateful if it is designed to remember preceding events or user interactions; the remembered information is called the state of the system.

https://en.wikipedia.org/wiki/State_(computer_science)

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (dlvenable) [reply _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md]: I believe either `SSL/TLS` or `TLS/SSL` is in use. I intentially chose `TLS/SSL` because we are using TLS. The SSL part is mostly there for historical reasons.

You can also see that the term `TLS/SSL` is used in the following Wikipedia article.

https://en.wikipedia.org/wiki/Transport_Layer_Security

**Review by dlvenable — COMMENTED** [review]
I pushed all the changes except the final call-to-action section. I will push that soon.

**Comment by dlvenable** [comment]
> @dlvenable I changed to "states" to match @Naarcha-AWS edits and also to avoid "all the state". If you want to use "state", remove "all".

That makes sense. I've removed "all" from the sentence.

**Comment by dlvenable** [comment]
I have also pushed a short conclusion section.

File: _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md
Comment (natebower) [inline _posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md:172]: Because this is a blog, an exclamation point would work after the first sentence. Other than that small nit, LGTM.

**Review by natebower — DISMISSED** [review]
LGTM

**Review by krisfreedain — APPROVED** [review]
looks good!

**Pull request merged**
